### ISSUE TITLE
Need to override jvm options on command line with 'run' command

### ISSUE THREAD

**Issue by countfloyd** [oyvindberg/bleep#361]
I have some -Dkey=value properties I'd like to pass when using the run command. Putting them in the bleep.yaml file works but there are some properties that are too sensitive to put in the build file and others that need to change for different invocations. I've tried:

bleep run project -Dkey=value
JAVA_OPTS="-Dkey=value" bleep run project

Can't see a way to do it.

Thanks!

**Comment by oyvindberg** [comment]
Yeah, likely not at this point. I think it'll be easy to add though if anybody wants to try. 

for now environment variables and parameters should work

**Comment by oyvindberg** [comment]
envvars are threaded through now
